config/kcp/build-service-apibinding.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIBinding
+metadata:
+  name: build-service-apibinding
+spec:
+  reference:
+    workspace:
+      exportName: build-service-apiexport
+      path: root:appstudio:build-service
+  permissionClaims:
+  - group: ""
+    resource: configmaps
+    state: Accepted
+  - group: ""
+    resource: secrets
+    state: Accepted
+  - group:
```

Comment:
curious that for some of the core k8s groups you have `""` and others you have just a blank

is that by design?

if so, I wonder if the reason why pertains to a problem I currently have where I can get some core k8s items listed in my permission claim, but not others

Reply:
There is no special meaning AFAIK. Added `""` for consistency.

Reply:
consistency sounds good to me